tests/_utils/logs.py

```
@@ -0,0 +1,20 @@
+import json
+import os
+
+
+def clear_logs() -> None:
+    """Find and clear the log file."""
+    try:
+        log_dir = os.path.dirname(__file__)
+        config = json.loads(open(os.path.join(log_dir,"test_config.json")).read())
```

Ruff

Use a context manager for opening files

(SIM115)